### PR TITLE
Update readme contents

### LINKED ISSUE #14: Update README.md
* Add CodeCov and Github Actions build status badges to the page
* Update contents to match InterviewHub
* Add acknowledgement for AB3

### PR DESCRIPTION
**Pull request by kiwibang**
- This PR edits the readme.md file of the team to suit our project 
- Please let me know if there are any changes that have to be made!
- Fixes #14 
🤞

### CODE REVIEW

**Comment by jingjie88** [comment]
LGTM!

File: README.md [hunk 1]
```diff
@@ -3,8 +3,20 @@
...
 ![Ui](docs/images/Ui.png)
 
-* Placeholder text
+* This CLI application is designed for **Engineering Managers of tech companies.**
+
+    Example usages:
+  * Schedule an interview with an interviewee
+  * Detect clashes in interview timings
+  * Write notes about the interviewee
 * For the detailed documentation of this project, see the **[InterviewHub Product Website](https://se-education.org/addressbook-level3)**.
```
Comment (ghost) [inline README.md]: The hyperlink destination should be updated to 
`https://ay2324s1-cs2103t-t11-2.github.io/tp/`

**Review by ghost — CHANGES_REQUESTED** [review]
LGTM, Just a small nit to be fixed

File: README.md (hunk 1 above)
Comment (kiwibang) [reply README.md]: - Yep, my apologies!

**Review by ghost — APPROVED** [review]

**Pull request merged**